specification/yaml/openapi_v2024-01-01.yml

```
@@ -1417,7 +1422,7 @@ paths:
         - Batch operations
       summary: Perform batch operations
       description: |
-        Performs multiple operations in a batch with a single request. You can include a maximum of 10 operations in a single batch request.
+        Performs multiple operations in a batch with a single request. You can include a maximum of 10 operations in a single batch request. For batch requests, the REST API rate limits support 500 burst requests, 100 maximum queue length, and 1 batch request per second.
```

As with the other PR

```suggestion
        Performs multiple operations in a batch with a single request. You can include a maximum of 10 operations in a single batch request. For batch requests, the REST API rate limits support 500 burst requests, a maximum of 100 requests in the queue, and 1 batch request per second.
```